app/main/views/direct_award.py

```diff
@@ -201,7 +201,60 @@ def get_project_search(project_id, search_id):
 
 @main.route('/direct-award/projects/<int:project_id>/services', methods=['GET'])
 def list_project_services(project_id):
-    raise NotImplementedError()
+    """This endpoint returns all the services associated with a particular (locked) Direct Award Project. It returns
+    some fairly arbitrary data which is obviously not ideal, but as we don't have a task runner that we can utilise to
+    manage jobs like this and we need it to be responsive on-click, we're having to jerryrig in some data extraction and
+    transformation in here. Specifically we are returning supplier name and contact information, and returning from
+    the service data JSON keys as specified in the request (which will be loaded via a framework-specific manifest).
+
+    TODO: If/When we get a DM task runner, this should be updated to just return a list of service IDs and the runner
+    becomes responsible for querying the API to get information on those services."""
+    page = get_valid_page_or_1()
+    requested_fields = request.args.get('fields', '').split(',')
+    project = DirectAwardProject.query.filter(DirectAwardProject.id == project_id).first_or_404()
+
+    if not project.locked_at:
+        abort(400, 'Project has not been locked: {}'.format(project_id))
+
+    search = DirectAwardSearch.query.filter(
+        DirectAwardSearch.project_id == project_id,
+        DirectAwardSearch.active == True  # noqa
+    ).first()
```

Comment:
FWIW I think this should be the combination of results from all active searches. The fact that only one search can be active at a time would be better managed in just one place? (i.e. no assumption made here)

Reply:
I was going to do that initially and decided against it... 🙈

Reply:
okay maybe we can just add a to-do in then